rules/aip0133/request_required_fields.go

```diff
@@ -26,19 +26,25 @@ import (
 )
 
 // The create request message should not have unrecognized fields.
-var requestRequiredFields = &lint.MessageRule{
+var requestRequiredFields = &lint.MethodRule{
 	Name:   lint.NewRuleName(133, "request-required-fields"),
-	OnlyIf: isCreateRequestMessage,
-	LintMessage: func(m *desc.MessageDescriptor) (problems []lint.Problem) {
-		resourceMsgName := getResourceMsgNameFromReq(m)
+	OnlyIf: utils.IsCreateMethod,
+	LintMethod: func(m *desc.MethodDescriptor) []lint.Problem {
+		ot := utils.GetResponseType(m)
+		r := utils.GetResource(ot)
+		resourceMsgName := ""
+		if r != nil {
+			resourceMsgName = utils.GetResourceSingular(r)
+		}
```

Comment:
I would just handle `nil` in `GetResourceSingular`, but this is fine for now.

Reply:
done!